compute_tools/Cargo.toml

```diff
@@ -15,12 +15,14 @@ aws-config.workspace = true
 aws-sdk-s3.workspace = true
 aws-sdk-kms.workspace = true
 anyhow.workspace = true
+axum = { workspace = true, features = [] }
 camino.workspace = true
 chrono.workspace = true
 cfg-if.workspace = true
 clap.workspace = true
 flate2.workspace = true
 futures.workspace = true
+http.workspace = true
 hyper0 = { workspace = true, features = ["full"] }
```

Comment:
this dependency is not needed any more, right?

Reply:
Have you tried removing it?

I don't see any of the code using it any more. And if it would, you'd likely get an error anyways because axum uses a newer version of hyper than the one we depend on here.

Reply:
Ah that's what you were talking about. Yes, I will remove it. Thanks for noticing that.